products/angular-js.md

```diff
@@ -0,0 +1,137 @@
+---
+title: AngularJS
+category: framework
+tags: javascript javascript-runtime
+iconSlug: angularjs
+permalink: /angularjs
+alternate_urls:
+-   /angular-js
+versionCommand: npm list angular
+releasePolicyLink: https://docs.angularjs.org/misc/version-support-status
+
+eolColumn: false
+activeSupportColumn: Community Support
+releaseColumn: Latest
+releaseDateColumn: Released
+extendedSupportColumn: Extended Long Term Support
+
+auto:
+  - npm: angular
+
+identifiers:
+  - repology: angular.js
+  - purl: pkg:npm/angular
+
+releases:
+-   releaseCycle: "1.8"
+    releaseDate: 2020-06-04
+    support: 2021-12-31
```

Comment:
I'm confused with this one.
If I removed `support` in favor of `eol` it would render in what I consider something misleading.

It would render on the Community Support column a big green __yes__ instead of indicating when the community support ended.

I believe that `support` and `eol` options should be there to render a red block indicating when the support finished.

![image](https://github.com/endoflife-date/endoflife.date/assets/7026066/cf6bed2a-44f3-4407-9853-bccdc75b56de)

Reply:
Solved on [05318d3](https://github.com/endoflife-date/endoflife.date/commit/05318d3cd0f0ef0b1c546700811665cb8995cc57)